core/spec/models/spree/line_item_spec.rb

```diff
@@ -13,17 +13,19 @@
     end
   end
 
-  context '#destroy' do
-    it "fetches deleted products" do
-      line_item.product.destroy
+  context '#discontinued' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.